clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/RecoveryTests.scala

```diff
@@ -0,0 +1,156 @@
+package org.broadinstitute.clio.integrationtest.tests
+
+import java.nio.file.Files
+import java.time.OffsetDateTime
+import java.util.Comparator
+
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import com.dimafeng.testcontainers.ForAllTestContainer
+import com.sksamuel.elastic4s.circe._
+import io.circe.Encoder
+import org.broadinstitute.clio.client.commands.ClioCommand
+import org.broadinstitute.clio.integrationtest.BaseIntegrationSpec
+import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
+  DocumentGvcf,
+  DocumentWgsUbam,
+  Elastic4sAutoDerivation,
+  ElasticsearchIndex
+}
+import org.broadinstitute.clio.server.dataaccess.util.ClioUUIDGenerator
+import org.broadinstitute.clio.transfer.model.{
+  TransferGvcfV1QueryOutput,
+  TransferWgsUbamV1QueryOutput
+}
+import org.broadinstitute.clio.util.model.{DocumentStatus, Location}
+import org.scalatest.{Args, Status}
+
+import scala.util.Random
+
+/** Tests of Clio's recovery mechanisms. */
+trait RecoveryTests extends ForAllTestContainer {
+  self: BaseIntegrationSpec =>
+
+  val documentCount = 10
```

Comment:
Would be nice to use > 10 documents. Using 50 causes the test startup to time out.

Reply:
Maybe the integration test could use a larger set.

Reply:
This is the integration test. I think I can get a larger number working when I recover ubams and gvcfs in parallel.